README.md

```diff
@@ -1,7 +1,5 @@
 # Linphone on iPhone
 
-[![Build Status](https://travis-ci.org/BelledonneCommunications/linphone-iphone.svg?branch=master)](https://travis-ci.org/BelledonneCommunications/linphone-iphone)
```

Comment:
This begs the question of whether we really want to start forking our codebase that far from the linphone mainstream codebase. I see where you are going here, but this kind of change will likely get a merge reject if we PR back upstream. I'll talk about this with Peter.